### ISSUE TITLE
HTMLDOMPropertyConfig no longer part of React core

### ISSUE THREAD

**Issue by cjke** [reactjs/react-magic#95]
In htmltojsx, there is the following line: 

`require('react/lib/HTMLDOMPropertyConfig')`

But, your package file only pulls react, not react-dom, which is causing errors such as:

```
module.js:327
    throw err;
    ^

Error: Cannot find module 'react/lib/HTMLDOMPropertyConfig'
```

This is only happening in the htmltojsx submodule (not the core package). I believe in your core package you are picking React ~14, whereas the submodule is picking React * (which in turn is grabbing 15.x).

**Comment by Daniel15** [comment]
Fixed by #97

**Comment by cjke** [comment]
Awesome - cheers!
